lib/katello_deploy/repositories.rb

```diff
@@ -20,17 +21,19 @@ def configure(koji_repos = false)
       bootstrap_puppet(@os_version)
 
       if koji_repos
-        setup_koji_repos(@os_version, @katello_version, foreman_version)
+        setup_foreman_koji_repos(@os_version, @version)
+        setup_katello_koji_repos(@os_version, katello_version) if @scenario == 'katello'
       else
-        bootstrap_katello(@katello_version, @os_version)
-        bootstrap_foreman(foreman_version, @os_version)
+        bootstrap_foreman(@version, @os_version)
+        bootstrap_katello(katello_version, @os_version) if @scenario == 'katello'
```

Comment:
We may need to update to include if scenario is 'katello' or 'devel'.  Currently, for the centos7-devel the katello koji repos are not getting included.